x-pack/plugins/ml/public/index.scss

```diff
@@ -50,3 +50,9 @@
   // Hacks are last so they can overwrite anything above if needed
   @import 'hacks';
 }
+// @import '~@elastic/eui/src/themes/eui/eui_colors_light';
+// @import '~@elastic/eui/src/global_styling/functions/index';
+// @import '~@elastic/eui/src/global_styling/variables/index';
+// @import '~@elastic/eui/src/global_styling/mixins/index';
+// @import '~@elastic/eui/src/global_styling/reset/index';
+// @import '~@elastic/charts/src/theme_only_light';
```

Comment:
Are these no longer needed?

Reply:
nope, left in by mistake. Removed in 6b94e24bd79bd7a14f74fb4ce57cf68dc2bfb55f